### ISSUE TITLE
Status Code Registry (Trac #59)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#59]
The IANA status code registry should be referred to.

Migrated from https://trac.ietf.org/ticket/59
```json
{
    "status": "closed",
    "changetime": "2008-06-12T09:57:08",
    "_ts": "1213264628000000",
    "description": "The IANA status code registry should be referred to.",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2007-12-22T02:41:55",
    "component": "p2-semantics",
    "summary": "Status Code Registry",
    "priority": "",
    "keywords": "",
    "milestone": "03",
    "owner": "julian.reschke@gmx.de",
    "type": "editorial",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20080104T06:06:25: 
* mnot@pobox.com changed component from "" to "non-specific"
* mnot@pobox.com changed milestone from "" to "unassigned"
* mnot@pobox.com changed owner from "" to ""

**Comment by mnot** [comment]
Trac update at 20080105T01:54:38: fielding@gbiv.com changed milestone from "unassigned" to "02"

**Comment by mnot** [comment]
Trac update at 20080228T05:26:41: mnot@pobox.com changed milestone from "02" to "03"

**Comment by mnot** [comment]
Trac update at 20080601T08:46:00: 
* julian.reschke@gmx.de changed component from "non-specific" to "p2-semantics"
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"

**Comment by mnot** [comment]
Trac update at 20080601T08:57:48: julian.reschke@gmx.de commented:

> From [255]:
> 
> Add IANA re-registration of status codes, referencing RFC 2817; todo: consider taking over the registry procedure (related to #59)

**Comment by mnot** [comment]
Trac update at 20080601T09:01:26: julian.reschke@gmx.de commented:

> I think we should also take over the registration procedure, obsoleting RFC2817, Section 7.1.

**Comment by mnot** [comment]
Trac update at 20080601T09:03:57: julian.reschke@gmx.de commented:

> From [256]:
> 
> Add missing change log entry (related to #59)

**Comment by mnot** [comment]
Trac update at 20080609T16:30:31: 
* julian.reschke@gmx.de changed attachment from "" to "i59.diff"
* julian.reschke@gmx.de commented:

> proposed change for part 2.

**Comment by mnot** [comment]
Trac update at 20080610T13:59:46: 
* julian.reschke@gmx.de changed attachment from "" to "i59.2.diff"
* julian.reschke@gmx.de commented:

> proposed change for part 2.

**Comment by mnot** [comment]
Trac update at 20080612T09:57:08: 
* julian.reschke@gmx.de commented:

> Fixed in [262]:
> 
> Resolve #59: Take over HTTP Status Code Registry from RFC2817 (closes #59).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "new" to "closed"
